CHANGELOG.md

```diff
@@ -1,3 +1,50 @@
+# 0.6.0
+
+## February 5, 2021
+
+This is a significant release of ACA-py with several new features, as well as changes to the internal architecture in order to set the groundwork for using the new shared component libraries: [indy-vdr](https://github.com/hyperledger/indy-vdr), [indy-credx](https://github.com/hyperledger/indy-shared-rs), and [aries-askar](https://github.com/hyperledger/aries-askar).
+
+### Mediator support
+
+While ACA-py had previous support for a basic routing protocol, this was never fully developed or used in practice. Starting with this release, inbound and outbound connections can be established through a mediator agent using the Aries (Mediator Coordination Protocol)[https://github.com/hyperledger/aries-rfcs/tree/master/features/0211-route-coordination]. This work was initially contributed by Adam Burdett and Daniel Bluhm on behalf of SICPA. [Read more about mediation support](./Mediation.md).
```

Comment:
Please change to "Adam Burdett and Daniel Bluhm of Indicio"

Reply:
...but still leave in the "on behalf of SICPA".  We should have links for SICPA -- https://www.sicpa.com/ and Indicio -- https://indicio.tech/